config/betamocks/services_config.yml

```diff
@@ -904,11 +903,14 @@
 
 # Lighthouse Benefits Documents API
 - :name: "Lighthouse Benefits Documents"
-  :base_uri: <%= "#{URI(Settings.lighthouse.benefits_documents.host).host}:#{URI(Settings.lighthouse.benefits_documents.host).port}" %>
+  :base_uri: <%= "#{URI(Settings.claims_api.benefits_documents.host).host}:#{URI(Settings.claims_api.benefits_documents.host).port}" %>
   :endpoints:
     - :method: :post
-      :path: "/benefits-documents/v1/documents"
-      :file_path: "/lighthouse/benefits_documents"
+      :path: "/services/benefits-documents/v1/documents"
+      :file_path: "/lighthouse/benefits_documents/upload"
+    - :method: :post
+      :path: "/services/benefits-documents/v1/documents/search"
+      :file_path: "/lighthouse/benefits_documents/search"
```

Comment:
this is manipulating someone's existing mock services. https://github.com/department-of-veterans-affairs/vets-api/pull/13090. Not sure how I didn't notice this before. 

I'm not sure how they would feel about us modifying the Settings value used. According to the settings files they do evaluate to the same value. Assuming each environment has parallel settings, but we would want to confirm if they want it to remain as `lighthouse` and/or vice versa. Just to make sure we're good to go.

may be worth checking in with the author of the previous PR if we want to do it this way. Just to confirm they're fine with the settings files being changed, and the responses we'll be sending in each of the calls. this could impact their tests / local dev as we would return a response body in the mockdata as of our changes, and it looks like they may just be bypassing as their were no files in the mockdata for the previous service.

Reply:
also, the upload response is incorrect in the mockdata repo.

https://developer.va.gov/explore/api/benefits-documents/docs?version=current

Reply:
@FonzMP   I moved the path to the files into the claims_api namespace here, reverted that change and also moved search into its own folder so others could be added down the line.